collections/src/main/scala/strawman/collection/convert/ImplicitConversions.scala

```diff
@@ -0,0 +1,172 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2006-2016, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://www.scala-lang.org/           **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+
+package strawman
+package collection
+package convert
+
+import java.{ lang => jl, util => ju }, java.util.{ concurrent => juc }
+import scala.Predef.String
```

Comment:
with the exception of this import, this file has exact the same code as https://github.com/scala/scala/blob/2.13.x/src/library/scala/collection/convert/ImplicitConversions.scala